pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -533,6 +533,12 @@ func (c *MaxPDVolumeCountChecker) predicate(pod *v1.Pod, meta PredicateMetadata,
 	}
 
 	numNewVolumes := len(newVolumes)
+
+	// quick return
+	if numNewVolumes == 0 {
```

Comment:
Delete comments or more detailed comments (returns early if no new volume is needed).

Reply:
Thanks @wgliang

Reply:
MaxPDVolumeCountChecker is deprecated, there is no reason to change its behavior.